vueConfig/connections.js

```diff
@@ -60,5 +60,8 @@ module.exports = [
   'ws://127.0.0.1:8178',
   'https://viewm.moonicorn.network:443',
   'https://localhost:1991', // server for bcvault
-  'https://rpc.atheios.org:443'
+  'https://rpc.atheios.org:443',
+  'wss://bridge.walletconnect.org:443',
+  'api.dex.ag:443',
+  'api-v2.dex.ag:443'
```

Comment:
this should be just `dexag.mewapi.io:443` there is no reason for api v1 I believe?